commands/video.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   name: featureName,
   run: async (context) => {
     if (/^win/.test(process.platform)) {
-      console.log('test');
+      context.print.debug('test');
```

Comment:
This should just probably be removed. I left it in there by accident.

Reply:
Removed that line.